packages/demo/public/options-ccp-demo.json

```diff
@@ -0,0 +1,305 @@
+            {
+                "title": "Patienten",
+                "dataKey": "patients"
+            }
+        ]
+    },
+    "backends": {
+        "spots": [
+            {
+                "name": "DKTK",
+                "backendMeasures": "DKTK_STRAT_DEF_IN_INITIAL_POPULATION",
+                "url": "https://backend.demo.lens.samply.de/prod/",
+                "sites": [
+                    "berlin",
+                    "dresden",
+                    "essen",
+                    "frankfurt",
+                    "freiburg",
+                    "hannover",
+                    "mainz",
+                    "muenchen-lmu",
+                    "muenchen-tum",
+                    "ulm",
+                    "wuerzburg",
+                    "mannheim",
+                    "dktk-test",
+                    "hamburg"
+                ],
+                "catalogueKeyToResponseKeyMap": [
+                    [
+                        "gender",
+                        "Gender"
+                    ],
+                    [
+                        "age_at_diagnosis",
+                        "Age"
+                    ],
+                    [
+                        "diagnosis",
+                        "diagnosis"
+                    ],
+                    [
+                        "medicationStatements",
+                        "MedicationType"
+                    ],
+                    [
+                        "sample_kind",
+                        "sample_kind"
+                    ],
+                    [
+                        "therapy_of_tumor",
+                        "ProcedureType"
+                    ],
+                    [
+                        "75186-7",
+                        "75186-7"
+                    ]
+                ]
+            }
+        ],
+        "blazes": [
+            {
+                "name": "DKTK",
+                "backendMeasures": "DKTK_STRAT_DEF_IN_INITIAL_POPULATION",
+                "url": "http://localhost:8080",
+                "catalogueKeyToResponseKeyMap": [
+                    [
+                        "gender",
+                        "Gender"
+                    ],
+                    [
+                        "age_at_diagnosis",
+                        "Age"
+                    ],
+                    [
+                        "diagnosis",
+                        "diagnosis"
+                    ],
+                    [
+                        "medicationStatements",
+                        "MedicationType"
+                    ],
+                    [
+                        "sample_kind",
+                        "sample_kind"
+                    ],
+                    [
+                        "therapy_of_tumor",
+                        "ProcedureType"
+                    ],
+                    [
+                        "75186-7",
+                        "75186-7"
+                    ]
+                ]
+            }
+        ]
```

Comment:
Why are you adding this to the demo project, which is intended to be hosted on demo.lens.samply.de? Wouldn't it make only sense on options-dev.json?

Reply:
As the backend url is now in the config, it's not possible to have these set by the environment in json, afaik. So we would have to distinguish between imported files rather than adding the string.

so far:
 const backendUrl =
        import.meta.env.VITE_TARGET_ENVIRONMENT === "production"
            ? "https://backend.data.dktk.dkfz.de/prod/"
            : "https://backend.demo.lens.samply.de/prod/";

new change:
  let optionsFilePath: string = "options-dev.json";

    if (import.meta.env.VITE_TARGET_ENVIRONMENT === "production") {
        optionsFilePath = "options-ccp-prod.json";
    } else if (import.meta.env.VITE_TARGET_ENVIRONMENT === "staging") {
        optionsFilePath = "options-ccp-demo.json";
    }

does that make sense, or is there a cleaner way?

Reply:
Sorry, i think i should have been more specific. The solution with the different config files per project is really good and i like that approach. I thought github would show the actual section i marked for my comment, but that is not the case. I actually wanted to ask why you add a blaze configuration for URL http://localhost:8080 to the production environment?